src/coreclr/vm/interpexec.cpp

```diff
@@ -21,6 +20,9 @@ FCDECL1(double, JIT_ULng2Dbl, uint64_t val);
 FCDECL1(float, JIT_Lng2Flt, int64_t val);
 FCDECL1(double, JIT_Lng2Dbl, int64_t val);
 
+#ifdef FEATURE_JIT
+#include "callstubgenerator.h"
```

Comment:
This should not be conditionalized on the FEATURE_JIT. It is needed for AOT too.